README.md

```diff
@@ -23,7 +23,6 @@ Then just clone the git repo and set up this app:
 git clone https://github.com/etesync/server.git etebase
```

Comment:
I'm not sure how much value there actually is in including the git clone part - I would probably just preface with clone the repository and go straight to setting up the `venv` (which was very appreciated for me, as someone that doesn't use python very often any more)

Reply:
It's just there to show people how to clone to a named repo as some were complaining it was just called `server`.